src/screens/Home/hooks/usePeople.ts

```diff
@@ -6,27 +6,36 @@ import { People } from '../types';
 
 interface UsePeople {
   people: People[];
+  isLoading: boolean;
+  isError: boolean;
 }
 
 const usePeople = (): UsePeople => {
   const [people, setPeople] = useState<People[]>([]);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [isError, setIsError] = useState<boolean>(false);
 
   useEffect(() => {
+    setIsLoading(true);
```

Comment:
```suggestion
    setIsLoading(true);
    setError(undefined);
```

a "good practice" es resetear el error cuando tratas de fetch otravez

Reply:
ok maybe ese era el error que tenia antes y por eso no funcionaba lol